### PR TITLE
Auto

### PR DESCRIPTION
**Pull request by DarthCooper**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Steps to Test
Run auto. It should run the shooters and then the intake shooting the ring.

## Changelog
Moved the default commands that use joystick input to teleop init so it doesn't fight auto.

## Why?
Believe it or not, we need an auto. This scores one piece and then does nothing.

### CODE REVIEW

File: robotswerve.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
 from commands.togglefielddrive import ToggleFieldDrive
 from commands.resetfielddrive import ResetFieldDrive
 
+from auto import SparkyShoot
+
 import math
```
Comment (ChrisHirsch) [inline robotswerve.py:30]:
```suggestion
```

Linter flagged this as unused https://github.com/Raptacon/Robot-2024/actions/runs/8073244520/job/22056564167?pr=59

File: robotswerve.py [hunk 2]
```diff
@@ -172,9 +174,9 @@ def testPeriodic(self) -> None:
         wheelAngle #"use" value
         wheelSpeed #"use" value
         self.driveTrain.getCurrentAngles()
-        LeftX = wpimath.applyDeadband(self.driveController.getLeftX(), 0.02)
-        LeftY = wpimath.applyDeadband(self.driveController.getLeftY(), 0.02)
-        RightY = wpimath.applyDeadband(self.driveController.getRightY(), 0.1)
+        #LeftX = wpimath.applyDeadband(self.driveController.getLeftX(), 0.02)
+        #LeftY = wpimath.applyDeadband(self.driveController.getLeftY(), 0.02)
+        #RightY = wpimath.applyDeadband(self.driveController.getRightY(), 0.1)
```
Comment (tuffinmuffin) [inline robotswerve.py:179]:
```suggestion
```

File: robotswerve.py [hunk 3]
```diff
@@ -186,19 +188,9 @@ def testPeriodic(self) -> None:
                 self.driveTrain.disable()
             case "Wheels Select":
                 #self.driveTrain.setSteer(wheelAngle)
```
Comment (tuffinmuffin) [inline robotswerve.py:190]:
```suggestion
```

File: subsystem/swerveDriveTrain.py [hunk 4]
```diff
@@ -85,12 +90,12 @@ def getHeading(self) -> Rotation2d:
...
+        self.resetOdometry()
 
     def drive(self, xSpeed: float, ySpeed: float, rot: float, fieldRelative: bool):
         #convert to proper units
         #actually don't
         rot = rot# * 180.0
-
         #print(f"drive: x {xSpeed}, y {ySpeed}, rot {rot}, field {fieldRelative}")
         #xSpeed = 0.0
         #ySpeed = 0.0
```
Comment (tuffinmuffin) [inline subsystem/swerveDriveTrain.py:101]:
```suggestion
```

**Review by ChrisHirsch — APPROVED** [review]
+1

**Review by tuffinmuffin — APPROVED** [review]
good enough for now...

**Review by ChrisHirsch — APPROVED** [review]
Good Enough for now!
+1Thumb

**Pull request merged**